### PR TITLE
Xopt geecs environments dev

### PR DESCRIPTION
**Pull request by skbarber**
badger/xopt interface seems in a good place. Chris made a nice super class for the geecs optimization environments with useful examples on how to set up different optimization problems. 

A basic workflow for installing and running everything from scratch on windows machine is provided in readme

A couple of small changes to geecs_python_api, which are very similar in nature to previous changes.

### CODE REVIEW

File: Xopt-GEECS/environment.yml [hunk 1]
```diff
@@ -1,11 +1,12 @@
-name: Xopt-GEECS
+name: Xopt-GEECS-test
```
Comment (TAU-Reinier) [inline Xopt-GEECS/environment.yml:1]: Is this still supposed to be Xopt-GEECS-test ?

**Review by TAU-Reinier — APPROVED** [review]
Looks good!

File: Xopt-GEECS/environment.yml (hunk 1 above)
Comment (skbarber) [reply Xopt-GEECS/environment.yml:1]: nope

Comment (TAU-Reinier) [reply Xopt-GEECS/environment.yml]: Alright, as a reminder any commits you tack on to this `xopt-geecs-environments-dev` branch, such as correcting this, will be automatically added to the pull request.

**Pull request merged**